src/components/VideoComponent.js

```diff
@@ -0,0 +1,66 @@
+/* Imports -------------------------------------------------------------------*/
+
+import Inferno from 'inferno';
+import Component from 'inferno-component';
+// import img from 'kalm-logo.png';
+
+/* Local variables -----------------------------------------------------------*/
+
+
+/* Component -----------------------------------------------------------------*/
+
+class VideoComponent extends Component {
+	constructor(props) {
+		super(props);
+	}
+
+	render() {
+		return (
+			<div>
+				<div className="jumbotron">
+					<div className="container">
+						<div className="row">
+							<div className="col-md-6 offset-md-3 text-center">
+								<img src="http://kalm.js.org/src/images/logo.png" />
+
+								<br />
+								<br />
```

Comment:
CSS styling is preferable to line breaks (`<br>`), which are more for paragraphs, ex: online article or book.